package.json

```diff
@@ -18,6 +18,8 @@
         "release": "lerna publish"
     },
     "dependencies": {
+        "@types/classnames": "^2.2.9",
```

Comment:
Dette kan vel være en devDependency? Ser for øvrig ut som `classnames` har typedefinisjoner, [men at de ikke blir eksportert med pakken ](https://github.com/JedWatson/classnames/issues/209)😑